packages/manager/apps/web/client/app/hosting/database/order-private/translations/Messages_fr_FR.json

```
@@ -0,0 +1,59 @@
+{
+  "hosting_database_order_private_title": "Commander une base de données SQL",
+  "hosting_database_order_private_description": "Vous devez avoir un hébergement pour commander les options Private SQL et Start SQL.",
+  "hosting_database_order_private_step_header_type": "Type de base de données",
+  "hosting_database_order_private_step_header_customize": "Personnalisation",
+  "hosting_database_order_private_step_header_duration": "Choisissez la durée",
+  "hosting_database_order_private_step_header_activation": "Activation",
+  "hosting_database_order_private_step_header_payment": "Paiement",
+  "hosting_database_order_private_step_description_activation": "Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation.",
+  "hosting_database_order_private_step_description_payment": "Afin de confirmer votre demande, veuillez choisir votre moyen de paiement, et accepter les conditions d'utilisation.",
+  "hosting_database_order_private_submit_activate": "Activer",
+  "hosting_database_order_private_submit_pay": "Payer",
+  "hosting_database_order_private_cancel": "Annuler",
+  "hosting_database_order_private_select_placeholder": "Sélectionner...",
+  "hosting_database_order_private_type": "Quelle offre de base de données désirez vous ?",
+  "hosting_database_order_private_type_label": "SQL Privé",
+  "hosting_database_order_private_type_text": "Bases de données aux performances mémoires (RAM) garanties pour votre hébergement Web-Hosting OVH",
```

If we refer to https://www.ovh.com/fr/
```suggestion
  "hosting_database_order_private_type_text": "Bases de données aux performances mémoires (RAM) garanties pour votre hébergement Web Hosting OVH",
```

---

Ok, I'll update it